### PR TITLE
Helm servicemonitor

### PR DESCRIPTION
**Pull request by arshamparity**
- ServiceMonitor for the telemetry-core service has been added. (Prometheus monitoring)
- tweaking the gitlab-ci to trigger automatically when new commit is pushed to the master branch.

### CODE REVIEW

File: helm/templates/frontend-nginx-configmap.yaml [hunk 1]
```diff
@@ -33,9 +33,5 @@ data:
           index index.html;
           listen 8000;
           server_name _;
-
-          location = /favicon.ico {
-            return 204;
-          }
```
Comment (jsdw) [inline helm/templates/frontend-nginx-configmap.yaml:39]: Github is showing this change, but this file doen't exist in master any more. Perhaps GitHub is just a bit slow?

Comment (jsdw) [reply helm/templates/frontend-nginx-configmap.yaml:39]: Ah, it's a conflict that needs resolving I suspect :)

**Review by jsdw — APPROVED** [review]
Looks great to me, just that wee conflict to fix :)

File: helm/templates/frontend-nginx-configmap.yaml (hunk 1 above)
Comment (arshamparity) [reply helm/templates/frontend-nginx-configmap.yaml:39]: yeah! I just removed them.

**Review by lovelaced — APPROVED** [review]
nice!

**Pull request merged**